### DISCUSSION TITLE
ERROR -Add users from AD group

### DISCUSSION THREAD

**Discussion by Puma4ka** [snipe/snipe-it#13410] [Q&A]
Good afternoon!
I'm trying to add users who are included in the active directory member group, but all the time it shows that
 
_**"Connection to LDAP was successful, however there were no users returned from your query. You should confirm the Base Bind DN above."**_

Can you please tell me what exactly is wrong and that it would be more correct to apply a filter where users are selected from a certain group and not just people in the container

![group](https://github.com/snipe/snipe-it/assets/66869511/01874de0-e778-4b21-8ef4-79c931131d3e)

**Comment by TShiremanLCOG** [comment]
I can't tell if the spaces (if there are any, difficult to tell) in your query might mess up the query. You can also test with Powershell (replace values as appropriate, or write your own query) using the following:

```
$ldap = Get-ADObject -LdapFilter "(&(objectCategory=Person)(objectClass=User)(primaryGroupID=513)(mail=*@whatever.com)(sAMAccountType=805306368)(!(userAccountControl:1.2.840.113556.1.4.803:=2)))" -Properties SAMAccountName,Description,department,company,DisplayName,Name,title,telephoneNumber -SearchBase "OU=OU_Users,DC=int,DC=whatever,DC=com"
ForEach-Object -InputObject $ldap {
    [PSCustomObject]@{
        SAMAccountName  = $_.SAMAccountName
        Description     = $_.Description
        Department      = $_.department
        Company         = $_.company
        DisplayName     = $_.DisplayName
        Name            = $_.Name
        Title           = $_.title
        TelephoneNumber = $_.telephoneNumber
    }
} 
$ldap | Export-Csv -Path c:\ldap.csv -NoTypeInformation

Write-Host "Done!"

**Comment by Puma4ka** [comment]
Thx
